.eslintrc.js

```diff
@@ -7,29 +7,21 @@ module.exports = {
     "sourceType": "module"
   },
   "env": {
-    "mocha": true,
-    "node": true,
-  },
-  "globals": {
-    "__CONFIG__": true,
-    "platform_exports": true,
-    "platform_require": true,
-    "assert": true,
-    "sinon": true
+    "node": true
```

Comment:
what do we need `node` for now that we use es modules?

Reply:
We have stuff in `bin/`, `{karma,webpack}.*, and various other files. Generally they lead to these errors:

```
'__dirname' is not defined  no-undef
'global' is not defined  no-undef
'module' is not defined   no-undef
'process' is not defined  no-undef
'require' is not defined  no-undef
```

Reply:
If you don't want to specify a global `env` of node, you could also specify it per-file with `/* eslint-env node */`, if it's just a handful of files (see https://eslint.org/docs/user-guide/configuring#specifying-environments).

Or, if it's a lot of deeply nested files [where more targeted .eslintrc.js files wouldn't work], it may be an excuse to use the new [Configuration Based on Glob Patterns](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns).

Reply:
I'll split out the removing node env from the top level in a separate bug. https://bugzilla.mozilla.org/show_bug.cgi?id=1429894